lib/python/test/testsuite/Math/deviation.py

```diff
@@ -0,0 +1,208 @@
+                 of the simulations are also a list, both must have
+                 the same length
+
+    simulation : float or array
+                 Values from the simulation
+                 Note: if the list and the values
+                 of the theory are also a list,
+                 both must have the same length
+
+    Return:
+    -------
+    out : The maximum in the deviation
+    """
+
+    difference = np.abs(theory - simulation)
+
+    return min(difference)
+
+
+def getDifference(theory, simulation):
+    """
+    calculates the difference between the theory value and
+    the simulation value
+
+    Calculates:
+    --------
+    theory -simulation
+
+    Input:
+    -------
+    theory :     float or array
+                 theoretical value
+                 Note: if list and the values
+                 of the simulations are also a list, both must have
+                 the same length
+
+    simulation : float or array
+                 Values from the simulation
+                 Note: if the list and the values
+                 of the theory are also a list,
+                 both must have the same length
+
+    Return:
+    -------
+    out : Array or float
+          the difference between the two values
+    """
+
+    return np.abs(theory - simulation)
+
+
+def getDifferenceInPercentage(theory, simulation):
+    """
+    calculates the difference between the theory value and
+    the simulation value in percentage
+
+    Calculates:
+    --------
+    (theory - simulation) / simulation * 100
+
+    Input:
+    -------
+    theory :     float or array
+                 theoretical value
+                 Note: if list and the values
+                 of the simulations are also a list, both must have
+                 the same length
+
+    simulation : float or array
+                 Values from the simulation
+                 Note: if the list and the values
+                 of the theory are also a list,
+                 both must have the same length
+
+    Return:
+    -------
+    out : Array or float
+          the difference between the two values in percentage
+
+    """
+
+    simMax = max(simulation)
+    return (theory - simMax) / theory * 100
+
+
+def getAcceptanceRange(theory, acceptance: float = None):
+    """
+    represents the range around the theoretical value,
+    which as a limit leads to the acceptance of the test
+
+    Input:
+    --------
+
+    theory :     float or array
+                 theoretical value
+
+    acceptance : float, optional
+                 acceptance percentage. If None, the value must have
+                 been specified in Data.py
+
+    Return:
+    -------
+    out : float or array
+    """
+
+    acceptance = cD.checkVariables(variable="acceptance",
+                                   parameter=acceptance)
+
+    return theory - (theory * acceptance), theory + (
+                       theory * acceptance)
```

Comment:
```suggestion
    return theory * (1  - acceptance), theory * (1 + acceptance)
```
Yours is correct but this might be shorter